recipe-server/client/control/components/MultiPicker.js

```diff
@@ -0,0 +1,234 @@
+import React, { PropTypes as pt } from 'react';
+
+export default class MultiPicker extends React.Component {
```

Comment:
Let me know if you need help or wanna pair on this. I've just recently used `FieldArray` so I'm currently a bit familiar with it.